File: exercism/python/etl/etl.py
# https://exercism.io/my/solutions/18c9c2b906624be88d7795c0da207542


def transform(legacy_data):
    new_data = dict()
    for k, v_lst in legacy_data.items():
        new_data.update({v.lower(): k for v in v_lst})
    return new_data

File: exercism/go/etl/etl.go
// https://exercism.io/my/solutions/e783f2c3fde94fba9096cad8df62dd66

package etl

import "strings"

// Transform takes a translation table in the form points->(letter, letter, ..)
// and returns a new one in the form letter->points
func Transform(scoreOld map[int][]string) map[string]int {
	scoreNew := make(map[string]int)
	for points, letters := range scoreOld {
		for _, letter := range letters {
			scoreNew[strings.ToLower(letter)] = points
		}
	}
	return scoreNew
}
